frontend/src/App.tsx

```diff
@@ -1,9 +1,10 @@
 import { useEffect, useState } from "react";
 import { getAllUsers } from "./requests/getAllUsers";
-import { userData } from "./requests/types";
+import { userData } from "./theme/types";
```

Comment:
Move back. Types for requests are not relevant to themes.

Reply:
done. not sure how that happened